app/assets/stylesheets/modules/mfa.css

```diff
@@ -7,6 +7,27 @@
   flex: 1;
 }
 
+.mfa__header-wrapper {
+  display: flex;
+  align-items: center;
+  gap: 20px;
+}
+
+.t-body .mfa__header {
+  display: inline-block;
+  padding-top: 0px;
+}
+
+.badge {
```

Comment:
We should create a new file for this `app/assets/stylesheets/modules/badge.css`. Conceptually, we should treat this as its own standalone component that can be used elsewhere.

Another place this class could live is in the `shared.css` file. I'm less enthusiastic that practice since it's less organized.

Lastly, we should add the `display: inline;` rule to this class. 

You may be wondering why -- 
In our present use-case, we don't need the `inline` rule on the badge since the layout concerns (e.g. the badge sitting side-by-side with the header) is taken care of by the parent element through the `flex` rule. However, if we treat the `.badge` as an isolated component responsible for its own concern, by default we'd want badges to be inline. If someone applies this class on a `div`, we'd want the `div` to have `inline` behaviour.